tensorflow/python/keras/activations.py

```diff
@@ -120,7 +120,7 @@ def selu(x):
   and the number of inputs is "large enough"
   (see references for more information).
 
-  ![](https://cdn-images-1.medium.com/max/1600/1*m0e8lZU_Zrkh4ESfQkY2Pw.png)
+  https://cdn-images-1.medium.com/max/1600/1*m0e8lZU_Zrkh4ESfQkY2Pw.png
```

Comment:
Don't remove the markdown markers here. They are useful to make the image show up as an image on the documentation page